index.js

```diff
@@ -68,7 +68,8 @@ Deployer.prototype.isReady = function(version) {
   return Promise.all(deferreds);
 };
 
-Deployer.prototype.deploy = function(version) {
+// imagePath is optional, and can be carried through the function calls as 'undefined'
+Deployer.prototype.deploy = function(version, imagePath) {
```

Comment:
I didn't realize that this was the deploy method that is actually called.
I've added the imagePath parameter here and to the internal deploy functional call (line 81) as well.